README.md

```diff
@@ -114,23 +114,21 @@ An overview of the project folders can be seen below.
-|/src/neo/Cryptography/|General cryptography classes including ECC implementation.|
-|/src/neo/IO/|Data structures used for caching and collection interaction.|
-|/src/neo/Ledger/|Classes responsible for the state control, including the `MemoryPool` and `Blockchain` classes.|
-|/src/neo/Network/|Peer-to-peer protocol implementation classes.|
-|/src/neo/Persistence/|Classes used to allow other classes to access application state.|
-|/src/neo/Plugins/|Interfaces used to extend Neo, including the storage interface.|
-|/src/neo/SmartContract/|Native contracts, `ApplicationEngine`, `InteropService` and other smart-contract related classes.|
-|/src/neo/VM/|Helper methods used to interact with the VM.|
-|/src/neo/Wallet/|Wallet and account implementation.|
-|/src/Neo.CLI/| `neo-cli` console application for the running a _FULL_ node.|
-|/src/Neo.Extensions/| Extensions class to expand the existing functionality.|
-|/src/Neo.Json/| Neo's JSON Specification classes.|
-|/src/Neo.VM/|Neo base class virtual machine.|
-|/src/plugins/| Include `ApplicationLogs`, `LevelDBStore`, `RpcServer`, `OracleService`, `DBFTPlugin` and more.|
-|/tests/|All unit tests.|
-
+|[/src/neo/Cryptography/](https://github.com/neo-project/neo/tree/master/src/Neo/Cryptography)|General cryptography classes including ECC implementation.|
+|[/src/neo/IO/](https://github.com/neo-project/neo/tree/master/src/Neo/IO)|Data structures used for caching and collection interaction.|
+|[/src/neo/Ledger/](https://github.com/neo-project/neo/tree/master/src/Neo/Ledger)|Classes responsible for the state control, including the `MemoryPool` and `Blockchain`.|
+|[/src/neo/Network/](https://github.com/neo-project/neo/tree/master/src/Neo/Network)|Peer-to-peer protocol implementation.|
+|[/src/neo/Persistence/](https://github.com/neo-project/neo/tree/master/src/Neo/Persistence)|Classes used to allow other classes to access application state.|
+|[/src/neo/Plugins/](https://github.com/neo-project/neo/tree/master/src/Neo/Plugins)|Interfaces used to extend Neo, including the storage interface.|
+|[/src/neo/SmartContract/](https://github.com/neo-project/neo/tree/master/src/Neo/SmartContract)|Native contracts, `ApplicationEngine`, `InteropService` and other smart-contract related classes.|
+|[/src/neo/VM/](https://github.com/neo-project/neo/tree/master/src/Neo/VM)|Helper methods used to interact with the VM.|
+|[/src/neo/Wallets/](https://github.com/neo-project/neo/tree/master/src/Neo/Wallets)|Wallet and account implementation.|
+|[/src/Neo.CLI/](https://github.com/neo-project/neo/tree/master/src/Neo.CLI)| `neo-cli` console application for the running a _FULL_ node.|
+|[/src/Neo.Extensions/](https://github.com/neo-project/neo/tree/master/src/Neo.Extensions)| Extensions to expand the existing functionality.|
+|[/src/Neo.Json/](https://github.com/neo-project/neo/tree/master/src/Neo.Json)| Neo's JSON specification.|
```

Comment:
More of implementation to me.

Reply:
It doesn't match any RFC paper, it's more to me as an specification or halfed implementation. But I see your point of view.